api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -218,6 +218,12 @@ message Permission {
     // Please refer to :ref:`this FAQ entry <faq_how_to_setup_sni>` to learn to
     // setup SNI.
     type.matcher.v3.StringMatcher requested_server_name = 9;
+
+    // A CIDR block that describes the upstream IP. This field is only applicable if the request is routed
```

Comment:
Nit: A CIDR block that will be used to match the upstream IP

Reply:
👍